extension/src/popup/components/sendPayment/SendConfirm/SubmitResult/styles.scss

```diff
@@ -47,14 +47,21 @@
     border-radius: 100px;
     padding: 0.25rem 0.5rem;
 
-    img {
-      width: 1rem;
-      height: 1rem;
+    .AccountAssets__asset--logo {
+      display: flex;
+      width: 0.875rem;
+      height: 0.875rem;
       border-radius: 1rem;
-      background: var(--pal-brand-primary-on);
       margin-right: 0.5rem;
     }
 
+    .AccountAssets__asset--error {
+      img {
+        width: 0.4375rem;
+        height: 0.4375rem;
+      }
+    }
+
```

Comment:
I went through and adjusted styles in the Senf/Swap flow that didn't play nice with the new structure. Let me know if anything looks crazy!